pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -141,7 +141,7 @@ func (r *ReconcileComplianceSuite) Reconcile(request reconcile.Request) (reconci
 	return res, nil
 }
 
-func (r *ReconcileComplianceSuite) suiteDeleteHandler(suite *compv1alpha1.ComplianceSuite, logger logr.Logger) error {
+func (r *ReconcileComplianceSuite) suiteDeleteHandler(suite *compv1alpha1.ComplianceSuite) error {
```

Comment:
instead of removing these parameters, I rather issue logs.

Reply:
OK, this change doesn't belong to the patch anyway

Reply:
Removed